app/models.py

```diff
@@ -72,11 +72,28 @@ class Post(PostData):
     created_at: datetime
     comments: List[Comment]
 
-class Event(BaseModel):
+class EventInput(BaseModel):
     title: str
-    description: Optional[str]
     date: datetime
     address: str
+    price: int
+    description: str # short info about the event
+    duration: Optional[int] # in hours
+    extraInformation: Optional[str] # more detailed practical information
+    maxParticipants: Optional[int]
```

Comment:
snake_case? since it's the convention in python and used other places

Reply:
Should probably be changed, but then the frontend should also be changed :/